src/Adafruit_DRV2605.cpp

```diff
@@ -0,0 +1,133 @@
+/*************************************************** 
```

Comment:
This looks like a library file. However it's located in the src folder. Can this file be moved?

Reply:
Moved the vibramotor library and the private libraries to the lib folder. Tested the features afterwards, still working.